besu/src/main/java/org/hyperledger/besu/cli/ConfigurationOverviewBuilder.java

```diff
@@ -142,13 +143,24 @@ public ConfigurationOverviewBuilder setEngineApis(final Collection<String> engin
   /**
    * Sets high spec enabled.
    *
-   * @return the high spec enabled
+   * @return the builder
    */
   public ConfigurationOverviewBuilder setHighSpecEnabled() {
     isHighSpec = true;
     return this;
   }
 
+  /**
+   * Sets the engine jwt file path.
+   *
+   * @param engineJwtFilePath the engine apis
+   * @return the builder
+   */
+  public ConfigurationOverviewBuilder setEngineJwtFile(final String engineJwtFilePath) {
```

Comment:
nit: could use Path or File here to make this even safer, e.g. if somehow the secret token ended up getting passed in by mistake